templates/taxi/car_confirm_delete.html

```diff
@@ -1,10 +1,11 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Delete car?</h1>
+  <h3>Are you sure, you want to delete car "{{ car.model }}"?</h3>
   <form action="" method="post">
     {% csrf_token %}
 
     <input type="submit" value="Yes" class="btn btn-danger">
+    <a href="{% if object.pk %}{% url 'taxi:car_detail' pk=object.pk %}{% else %}{% url 'taxi:car_list' %}{% endif %}" class="btn btn-secondary">Cancel</a>
```

Comment:
This is a code style issue. The `Cancel` button should lead to the previous page the user was on. Please, make sure to implement this logic. You can use the Django messages framework to pass the previous page URL to the template.